README.md

```diff
@@ -78,4 +79,8 @@ brew install gnu-sed
 After this, alter PATH. For example, add the following line to your `~/.bash_profile`:
 ```
 export PATH="/opt/homebrew/opt/gnu-sed/libexec/gnubin:$PATH"
-```
\ No newline at end of file
+```
+
+### TechDocs Images
+
+If you encounter images within TechDocs not properly rendering on Red Hat Developer Hub you may need to ensure they are located in a directory named `/images`.
```

Comment:
```suggestion
If you encounter images within TechDocs not properly rendering on Red Hat Developer Hub you may need to ensure they are located in a directory named `content/docs/images` in your template's dir.
```

Reply:
During `generate` it copies `/docs` from `skeleton` which already contains `images`, see https://github.com/Jdubrick/ai-lab-template/blob/fix-techdoc-images/scripts/import-ai-lab-samples#L36. If we created the dir it will duplicate

Reply:
Yeah that's correct too. We could maybe check that on the samples repo side.

Reply:
I don't believe the samples repo contains techdocs, they're all added in post when we pull them in

Reply:
Yes, that's correct.

Reply:
> I don't believe the samples repo contains techdocs, they're all added in post when we pull them in
Yeah probably my bad. Feel free to review the suggestion I made :)

Reply:
@thepetk please see my latest commit, I added a variation of your suggestion for clarity.